Manifests/ManagedPreferencesApplications/com.1password.1password.plist

```diff
@@ -0,0 +1,391 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_app_min</key>
+	<string>8.0</string>
+	<key>pfm_app_url</key>
+	<string>https://1password.com</string>
+	<key>pfm_description</key>
+	<string>1Password 8 settings</string>
```

Comment:
Thinking a bit longer term, since the preference domain has been standardized to not be version specific, perhaps this shouldn't be either.

Reply:
@apizz When I remove the version number, in ProfileCreator there are two entries with the same name. Is there a better way to distinguish between 1Password 7 and 1Password 8?


<img width="913" alt="Screenshot 2022-11-11 at 12 35 28 AM" src="https://user-images.githubusercontent.com/88400160/201290132-fc61202b-78aa-43ba-a6d1-8b2ccc3dd3a0.png">

Reply:
Fixed in [commit](https://github.com/ProfileCreator/ProfileManifests/pull/577/commits/3e56ce26a0e81a6d54a6bd2dc6540250952ffc40)